eng/pipelines/rush-update.yml

```diff
@@ -0,0 +1,28 @@
+trigger: none
```

Comment:
Can we please add this to an existing daily pipeline like aggregate-reports instead of creating another pipeline we need to monitor.